src/dashboard.php

```diff
@@ -26,8 +26,18 @@ function wp_lemme_know_dashboard_callback()
     $tableName = $wpdb->prefix . 'lemme_know_subscribers';
     $subscriberCount = $wpdb->get_var(sprintf('SELECT COUNT(*) FROM `%s`', $tableName));
 
+    $subscribers = $wpdb->get_results(sprintf("SELECT `s_email`, `s_date` FROM `%s` WHERE `s_confirmed`='yes' ORDER BY `s_date` DESC", $tableName));
+    $results = [];
+    foreach ($subscribers as $subscriber) {
+        $results[] = [
+            'email' => esc_html($subscriber->s_email),
```

Comment:
This is really nice improvement, thank you :)

Just one question, why do we need `esc_html()` for e-mail address? We already validate e-mails before inserting to database.

Reply:
If an email address passes `FILTER_VALID_EMAIL`, it may still contain [characters](https://en.wikipedia.org/wiki/Email_address#Local-part) that should be escaped when printing to HTML. 
To see this for yourself, execute this little PHP script (or use an [online](https://www.w3schools.com/php/phptryit.asp?filename=tryphp_compiler) compiler) and notice how the email address passes the test, yet causes the JavaScript to run:
```php
// Email to check
$email = '"john.doe<script>alert(\'hi\')</script>"@example.com';

// Validate email
if (filter_var($email, FILTER_VALIDATE_EMAIL)) {
  echo("$email is a valid email address");
} else {
  echo("$email is not a valid email address");
}
```

Reply:
I see. In that case I should also add extra validation when we insert email to database.